android/sdk/src/main/java/com/taobao/weex/ui/component/richtext/node/ANode.java

```diff
@@ -54,7 +56,7 @@ protected boolean isInternalNode() {
   protected void updateSpans(SpannableStringBuilder spannableStringBuilder, int level) {
     super.updateSpans(spannableStringBuilder, level);
     if (attr != null && attr.containsKey(HREF)) {
-      ASpan aSpan = new ASpan(mInstanceId, attr.get(HREF).toString());
+      ASpan aSpan = new ASpan(mInstanceId, attr.get(HREF).toString(), mComponentRef,WXUtils.getString(attr.get(RichTextNode.PSEUDO_REF),""));
```

Comment:
Why not just use `ItemClickSpan`?

Ref [ImgNode](https://github.com/apache/incubator-weex/blob/12e605fdec041e2a661edf45470815595f0fca2d/android/sdk/src/main/java/com/taobao/weex/ui/component/richtext/node/ImgNode.java#L78-L81) for detail.